core/server/worker/src/test/java/alluxio/worker/block/SpaceReserverTest.java

```diff
@@ -66,7 +65,6 @@ public void after() {
     mExecutorService.shutdownNow();
   }
 
-  @Ignore("https://alluxio.atlassian.net/browse/ALLUXIO-2908")
   @Test
   public void reserveCorrectAmountsOfSpace() throws Exception {
     BlockWorker blockWorker = PowerMockito.mock(BlockWorker.class);
```

Comment:
You will also need to change 
```java
public final class BlockStoreMeta
```
to
```java
public class BlockStoreMeta
```
otherwise, it will not compile